xbmc/settings/Settings.cpp

```diff
@@ -165,6 +165,7 @@ const std::string CSettings::SETTING_SUBTITLES_FONT = "subtitles.font";
 const std::string CSettings::SETTING_SUBTITLES_HEIGHT = "subtitles.height";
 const std::string CSettings::SETTING_SUBTITLES_STYLE = "subtitles.style";
 const std::string CSettings::SETTING_SUBTITLES_COLOR = "subtitles.color";
+const std::string CSettings::SETTING_SUBTITLES_OPACITY = "subtitles.opacity";
```

Comment:
can you move this into `OverlayRenderer.h` It can be included in the `OVERLAY` namespace as it doesn't need to be a general `CSetting`